public/stylesheets/style.css

```diff
@@ -39,6 +39,11 @@
         bottom: 0;
         right: 0;
     }
+
+
+    a#playlist{
```

Comment:
Have you turned off pointer for links in general because I would expect 'a' elements to already be pointers.